_posts/en/newsletters/2021-02-24-newsletter.md

```diff
@@ -0,0 +1,156 @@
+---
+title: 'Bitcoin Optech Newsletter #137'
+permalink: /en/newsletters/2021/02/24/
+name: 2021-02-24-newsletter
+slug: 2021-02-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes the results of discussion about
+choosing activation parameters for a taproot soft fork and includes our
+regular sections with selected questions and answers from the Bitcoin
+StackExchange, releases and release candidates, and notable changes to
+popular Bitcoin infrastructure software.
+
+## News
+
+- **Taproot activation discussion:** Michael Folkson [summarized][folkson
+  lot] a second meeting about activation parameters for [taproot][topic
+  taproot], concluding that "there wasn’t overwhelming consensus for
+  either LOT=true or LOT=false", the LockinOnTimeout (LOT) parameter
+  from [BIP8][] that determines whether or not nodes will require
+  mandatory signaling for activation of the fork.
+  However, there was nearly universal agreement on other
+  activation parameters, most notably reducing the amount of hash rate
+  that needs to signal for the fork to activate from 95% to 90%.
+
+    Discussion continued about the LOT parameter on the mailing list,
+    mainly about the effect of encouraging users to choose the option
+    themselves, either via a command line option or by choosing what
+    software release to use.  No clear agreement was
+    reached as of this writing and there does not seem to be a widely
+    supported path forward to activating taproot---even though taproot
+    itself appears to be almost entirely desired.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Is sharding a good alternative to multisig?]({{bse}}102007)
+  User S.O.S asks about the viability of a sharding scheme like Shamir’s Secret
+  Sharing (SSS) to achieve multisig-like capabilities. Pieter Wuille
+  notes the advantages of `OP_CHECKMULTISIG` over SSS (accountability, no need to
+  reconstruct the secret key on one machine), advantages of SSS over
+  `OP_CHECKMULTISIG` (less fees, more privacy), and [schnorr][topic schnorr signatures] considerations
```

Comment:
Perhaps 'schnorr considerations' -> 'alternative schemes if the schnorr/taproot soft fork is activated'